src/libDirectoryService/DSBlockPostProcessing.cpp

```diff
@@ -91,11 +91,12 @@ void DirectoryService::SendDSBlockToLookupNodes() {
 
   vector<unsigned char> dsblock_message = {MessageType::NODE,
                                            NodeInstructionType::DSBLOCK};
-  if (!Messenger::SetNodeDSBlock(dsblock_message, MessageOffset::BODY, 0,
-                                 *m_pendingDSBlock, m_shards, m_DSReceivers,
-                                 m_shardReceivers, m_shardSenders)) {
+  if (!Messenger::SetNodeVCDSBlocksMessage(
+          dsblock_message, MessageOffset::BODY, 0, *m_pendingDSBlock,
+          m_VCBlockVector, m_shards, m_DSReceivers, m_shardReceivers,
```

Comment:
Do we need to use the mutex here and elsewhere below for `m_VCBlockVector`? Just checking.